scss/_patterns_navigation.scss

```diff
@@ -652,6 +651,7 @@ $spv-navigation-logo: 0.3125rem;
 
   .p-navigation__logo-title {
     color: $color-navigation-text;
+    padding-left: $sph--small;
```

Comment:
Adjust padding of `p-navigation__link` above, in line 252, instead of adding padding to the title.